fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/typing/Type.java

```diff
@@ -511,7 +559,7 @@ enum TypeCode {
         STRING(String.class, FieldDescriptorProto.Type.TYPE_STRING, true, false),
         ENUM(Enum.class, FieldDescriptorProto.Type.TYPE_ENUM, false, false),
         RECORD(Message.class, null, false, false),
-        ARRAY(Array.class, null, false, false),
+        ARRAY(List.class, null, false, false),
```

Comment:
Yikes, thanks for fixing. Don't know what I was thinking.

Reply:
No worries